### DISCUSSION TITLE
Bulk delete in 0.15-beta-3

### DISCUSSION THREAD

**Discussion by maxx-ukoo** [blakeblackshear/frigate#15535] [Ask A Question] [answered]
How I can delete some videos in explore view?
I can press Ctrl-A to select all and delete them with delete button but how I can select only few videos in explore view?

**Comment by hawkeye217** [comment, marked as answer]
Right-click or long press on a tracked object to enter selection mode, then click or tap the ones you want. Just like the Review screen.

**Reply by maxx-ukoo** [reply]
Thanks it works from desktop and mobile
